.github/workflows/ci.yaml

```diff
@@ -13,24 +13,18 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest] # [macOS-latest, windows-latest] do not come with docker :(
-        cabal: ["3.2"]
+        cabal: ["3.6"]
         ghc:
-          - "8.8.3"
-          - "8.10.1"
-          - "8.10.2"
-        exclude:
```

Comment:
These excludes were pointless, as you don't test with macOS-latest and windows-latest anyway.